exercises/practice/word-count/test.scm

```diff
@@ -376,4 +198,3 @@
      (load "word-count.scm")
      (run-docker test-cases)]
     [else (load (cadr args)) (test 'input 'output)]))
-
```

Comment:
Hmm, your editor probably deleted the trailing newline.  This is noise.  I don't think we have a track policy regarding trailing newlines, and perhaps a policy would be contentious.  I don't mind merging as is, but want to make sure we don't end up with our editors fighting about trailing newlines and creating noise in commits.

FWIW, I have `require-final-newline` set to `nil` in my emacs, although I thought I had it otherwise.  I have aggressive trailing whitespace deletion on save, and I think in the past I had emacs insert a final trailing newline right before save to make it easier when obsessively spamming save while adding code to the end of a file.  Anyway, I have gotten pretty tangential here.  What is you ropinion about this @austinmlv ?  Do you think we need a policy?  Do you think it matters at all?  Let me know.

Reply:
I'm indifferent to the presence of trailing newlines. I do value consistency though. I probably deleted it out of habit from my personal projects. With that said, I restored it and I'm cool with having a code style / format policy, or something like an EditorConfig file. I use neovim and have the EditorConfig plugin installed so that may be an option. Alternatively maybe the project needs a `CONTRIBUTING` file?

Reply:
Yeah, we might need one (a CONTRIBUTING file).  For now I am rtrying some necromancy which includes a now functional-but-not-deployed reworking of the test-system to work with a new conformant test-runner.  Feel free to ponder what might be in our contributing docs.  We already have something from another maintainer written in org.  A decent amount of that info is about to be out of date though.